### ISSUE TITLE
Search-Elasticsearch-Async-6.80: test suite fails (with newer Search::Elasticsearch?)

### ISSUE THREAD

**Issue by eserte** [elastic/elasticsearch-perl#182]
The test suite fails on some of my smoker systems:
```
#   Failed test 'No body - format'
#   at t/30_Logger_Async/40_trace_request.t line 28.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true'
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1'
# '

#   Failed test 'Body - format'
#   at t/30_Logger_Async/40_trace_request.t line 48.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# {
#    "foo" : "bar\n\u0027baz"
# }
# '
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# {
#    "foo" : "bar\n\u0027baz"
# }
# '
# '

#   Failed test 'Bulk - format'
#   at t/30_Logger_Async/40_trace_request.t line 72.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# {"foo":"bar\n\u0027baz"}
# {"foo":"bar\n\u0027baz"}
# '
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# {"foo":"bar\n\u0027baz"}
# {"foo":"bar\n\u0027baz"}
# '
# '

#   Failed test 'Body string - format'
#   at t/30_Logger_Async/40_trace_request.t line 94.
#          got: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '
# The quick brown fox
# jumped over the lazy dog\u0027s basket'
# '
#     expected: '# Request to: https://foo.bar:444/some/path
# curl -H "Content-type: application/json" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '
# The quick brown fox
# jumped over the lazy dog\u0027s basket'
# '
# Looks like you failed 4 tests of 11.
t/30_Logger_Async/40_trace_request.t ................ 
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/11 subtests 
...
#   Failed test 'Default - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Host - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Host:Port - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'HTTPS - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Path - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'IPv4 - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Scheme:IPv4 - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'IPv4:Port - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Scheme:IPv4:Port - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'HTTPS option - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'HTTPS option with settings - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Port option - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Port option with settings - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Path option - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Path option with settings - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Deflate option - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'IPv4 with Port - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Hash host - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Hash port - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Hash path - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'

#   Failed test 'Hash IPv4 host - default_headers'
#   at t/60_Cxn_Async/30_http.t line 159.
# Comparing hash keys of $data
# Extra: 'User-Agent'
# Looks like you failed 21 tests of 133.
t/60_Cxn_Async/30_http.t ............................ 
Dubious, test returned 21 (wstat 5376, 0x1500)
Failed 21/133 subtests 
```
Statistical analysis suggests that failures happen with Search::Elasticsearch 6.80:
```
****************************************************************
Regression 'mod:Search::Elasticsearch'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	23050509021321440.00
[1='eq_6.80']  	     -1.0000	      0.0000	-17770426956505136.00

R^2= 1.000, N= 106, K= 2
****************************************************************
```

**Comment by ezimuel** [comment]
@eserte thanks for reporting! I'm investigating more, I can confirm that the `Extra: 'User-Agent'` error is because I added this header and I didn't update the async tests, I only updated the sync one. I'll fix it in the next patch release.

**Comment by jkeenan** [comment]
Similarly observed today on FreeBSD-12 against a perl near 5.32-RC0:
```
$ dumpjson EZIMUEL.Search-Elasticsearch-Async-6.80.log.json
{
  author => "EZIMUEL",
  dist => "Search-Elasticsearch-Async",
  distname => "Search-Elasticsearch-Async-6.80",
  distversion => "6.80",
  grade => "FAIL",
  prereqs => undef,
  test_output => [
    "Building and testing Search-Elasticsearch-Async-6.80",
    "cp lib/Search/Elasticsearch/CxnPool/Async/Sniff.pm blib/lib/Search/Elasticsearch/CxnPool/Async/Sniff.pm",
    "cp lib/Search/Elasticsearch/Async.pm blib/lib/Search/Elasticsearch/Async.pm",
    "cp lib/Search/Elasticsearch/Client/6_0/Async/Scroll.pm blib/lib/Search/Elasticsearch/Client/6_0/Async/Scroll.pm",
    "cp lib/Search/Elasticsearch/Cxn/AEHTTP.pm blib/lib/Search/Elasticsearch/Cxn/AEHTTP.pm",
    "cp lib/Search/Elasticsearch/CxnPool/Async/Static.pm blib/lib/Search/Elasticsearch/CxnPool/Async/Static.pm",
    "cp lib/Search/Elasticsearch/Cxn/Mojo.pm blib/lib/Search/Elasticsearch/Cxn/Mojo.pm",
    "cp lib/Search/Elasticsearch/Client/6_0/Async.pm blib/lib/Search/Elasticsearch/Client/6_0/Async.pm",
    "cp lib/Search/Elasticsearch/Role/Cxn/Async.pm blib/lib/Search/Elasticsearch/Role/Cxn/Async.pm",
    "cp lib/Search/Elasticsearch/CxnPool/Async/Static/NoPing.pm blib/lib/Search/Elasticsearch/CxnPool/Async/Static/NoPing.pm",
    "cp lib/Search/Elasticsearch/Client/6_0/Async/Bulk.pm blib/lib/Search/Elasticsearch/Client/6_0/Async/Bulk.pm",
    "cp lib/Search/Elasticsearch/Transport/Async.pm blib/lib/Search/Elasticsearch/Transport/Async.pm",
    "cp lib/Search/Elasticsearch/Async/Util.pm blib/lib/Search/Elasticsearch/Async/Util.pm",
    "cp lib/Search/Elasticsearch/Role/Is_Async.pm blib/lib/Search/Elasticsearch/Role/Is_Async.pm",
    "PERL_DL_NONLAZY=1 \"/usr/home/jkeenan/testing/smoke-me/khw-intrpvar/bin/perl\" \"-MExtUtils::Command::MM\" \"-MTest::Harness\" \"-e\" \"undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')\" t/*.t t/10_Basic_Async/*.t t/20_Serializer_Async/*.t t/30_Logger_Async/*.t t/40_Transport_Async/*.t t/50_Cxn_Pool_Async/*.t t/60_Cxn_Async/*.t t/Client_6_0_Async/*.t",
    "t/10_Basic_Async/10_load.t .......................... ok",
    "t/20_Serializer_Async/10_load_cpanel.t .............. ok",
    "t/20_Serializer_Async/11_load_xs.t .................. ok",
    "t/20_Serializer_Async/12_load_pp.t .................. ok",
    "t/20_Serializer_Async/13_preload_cpanel.t ........... ok",
    "t/20_Serializer_Async/14_preload_xs.t ............... ok",
    "t/20_Serializer_Async/20_xs_encode_decode.t ......... ok",
    "t/20_Serializer_Async/21_xs_encode_bulk.t ........... ok",
    "t/20_Serializer_Async/22_xs_encode_pretty.t ......... ok",
    "t/20_Serializer_Async/30_cpanel_encode_decode.t ..... ok",
    "t/20_Serializer_Async/31_cpanel_encode_bulk.t ....... ok",
    "t/20_Serializer_Async/32_cpanel_encode_pretty.t ..... ok",
    "t/20_Serializer_Async/40_pp_encode_decode.t ......... ok",
    "t/20_Serializer_Async/41_pp_encode_bulk.t ........... ok",
    "t/20_Serializer_Async/42_pp_encode_pretty.t ......... ok",
    "Current cxns: [\"http://localhost:9200\"]",
    "Forcing ping before next use on all live cxns",
    "Ping [http://localhost:9200] before next request",
    "t/30_Logger_Async/10_explicit.t ..................... ok",
    "Current cxns: [\"http://localhost:9200\"]",
    "Forcing ping before next use on all live cxns",
    "Ping [http://localhost:9200] before next request",
    "t/30_Logger_Async/20_implicit.t ..................... ok",
    "t/30_Logger_Async/30_log_methods.t .................. ok",
    "",
    "#   Failed test 'No body - format'",
    "#   at t/30_Logger_Async/40_trace_request.t line 28.",
    "#          got: '# Request to: https://foo.bar:444/some/path",
    "# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true'",
    "# '",
    "#     expected: '# Request to: https://foo.bar:444/some/path",
    "# curl -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1'",
    "# '",
    "",
    "#   Failed test 'Body - format'",
    "#   at t/30_Logger_Async/40_trace_request.t line 48.",
    "#          got: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '",
    "# {",
    "#    \"foo\" : \"bar\\n\\u0027baz\"",
    "# }",
    "# '",
    "# '",
    "#     expected: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '",
    "# {",
    "#    \"foo\" : \"bar\\n\\u0027baz\"",
    "# }",
    "# '",
    "# '",
    "",
    "#   Failed test 'Bulk - format'",
    "#   at t/30_Logger_Async/40_trace_request.t line 72.",
    "#          got: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '",
    "# {\"foo\":\"bar\\n\\u0027baz\"}",
    "# {\"foo\":\"bar\\n\\u0027baz\"}",
    "# '",
    "# '",
    "#     expected: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '",
    "# {\"foo\":\"bar\\n\\u0027baz\"}",
    "# {\"foo\":\"bar\\n\\u0027baz\"}",
    "# '",
    "# '",
    "",
    "#   Failed test 'Body string - format'",
    "#   at t/30_Logger_Async/40_trace_request.t line 94.",
    "#          got: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=true' -d '",
    "# The quick brown fox",
    "# jumped over the lazy dog\\u0027s basket'",
    "# '",
    "#     expected: '# Request to: https://foo.bar:444/some/path",
    "# curl -H \"Content-type: application/json\" -XPOST 'http://localhost:9200/xyz?foo=bar&pretty=1' -d '",
    "# The quick brown fox",
    "# jumped over the lazy dog\\u0027s basket'",
    "# '",
    "# Looks like you failed 4 tests of 11.",
    "t/30_Logger_Async/40_trace_request.t ................ ",
    "Dubious, test returned 4 (wstat 1024, 0x400)",
    "Failed 4/11 subtests ",
    "t/30_Logger_Async/50_trace_response.t ............... ok",
    "t/30_Logger_Async/60_trace_error.t .................. ok",
    "t/30_Logger_Async/70_trace_comment.t ................ ok",
    "t/30_Logger_Async/80_deprecation_methods.t .......... ok",
    "t/40_Transport_Async/10_tidy_request.t .............. ok",
    "t/40_Transport_Async/20_send_body_as.t .............. ok",
    "t/40_Transport_Async/30_perform_request.t ........... ok",
    "t/50_Cxn_Pool_Async/10_static_normal.t .............. ok",
    "t/50_Cxn_Pool_Async/11_static_node_missing.t ........ ok",
    "t/50_Cxn_Pool_Async/12_static_node_fails.t .......... ok",
    "t/50_Cxn_Pool_Async/13_static_node_timesout.t ....... ok",
    "t/50_Cxn_Pool_Async/14_static_both_nodes_timeout.t .. ok",
    "t/50_Cxn_Pool_Async/15_static_both_nodes_fail.t ..... ok",
    "t/50_Cxn_Pool_Async/16_static_nodes_starting.t ...... ok",
    "t/50_Cxn_Pool_Async/17_static_runaway_nodes.t ....... ok",
    "t/50_Cxn_Pool_Async/30_sniff_normal.t ............... ok",
    "t/50_Cxn_Pool_Async/31_sniff_new_nodes.t ............ ok",
    "t/50_Cxn_Pool_Async/32_sniff_node_fails.t ........... ok",
    "t/50_Cxn_Pool_Async/33_sniff_both_nodes_fail.t ...... ok",
    "t/50_Cxn_Pool_Async/34_sniff_node_timeout.t ......... ok",
    "t/50_Cxn_Pool_Async/35_sniff_both_nodes_timeout.t ... ok",
    "t/50_Cxn_Pool_Async/36_sniff_nodes_starting.t ....... ok",
    "t/50_Cxn_Pool_Async/37_sniff_runaway_nodes.t ........ ok",
    "t/50_Cxn_Pool_Async/38_bad_sniff.t .................. ok",
    "t/50_Cxn_Pool_Async/39_sniff_max_content.t .......... ok",
    "t/50_Cxn_Pool_Async/40_sniff_extract_host.t ......... ok",
    "t/50_Cxn_Pool_Async/50_noping_normal.t .............. ok",
    "t/50_Cxn_Pool_Async/51_noping_node_fails.t .......... ok",
    "t/50_Cxn_Pool_Async/52_noping_node_timesout.t ....... ok",
    "t/50_Cxn_Pool_Async/53_noping_all_nodes_fail.t ...... ok",
    "t/50_Cxn_Pool_Async/54_noping_nodes_starting.t ...... ok",
    "t/50_Cxn_Pool_Async/55_noping_runaway_nodes.t ....... ok",
    "t/50_Cxn_Pool_Async/56_max_retries.t ................ ok",
    "t/60_Cxn_Async/10_basic.t ........................... ok",
    "t/60_Cxn_Async/20_process_response.t ................ ok",
    "",
    "#   Failed test 'Default - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Host - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Host:Port - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'HTTPS - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Path - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'IPv4 - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Scheme:IPv4 - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'IPv4:Port - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Scheme:IPv4:Port - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'HTTPS option - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'HTTPS option with settings - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Port option - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Port option with settings - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Path option - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Path option with settings - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Deflate option - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'IPv4 with Port - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Hash host - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Hash port - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Hash path - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "",
    "#   Failed test 'Hash IPv4 host - default_headers'",
    "#   at t/60_Cxn_Async/30_http.t line 159.",
    "# Comparing hash keys of \$data",
    "# Extra: 'User-Agent'",
    "# Looks like you failed 21 tests of 133.",
    "t/60_Cxn_Async/30_http.t ............................ ",
    "Dubious, test returned 21 (wstat 5376, 0x1500)",
    "Failed 21/133 subtests ",
    "t/author-eol.t ...................................... skipped: these tests are for testing by the author",
    "t/author-no-tabs.t .................................. skipped: these tests are for testing by the author",
    "t/author-pod-syntax.t ............................... skipped: these tests are for testing by the author",
    "t/Client_6_0_Async/00_print_version.t ............... skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/10_live.t ........................ skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/15_conflict.t .................... skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/20_fork_aehttp.t ................. skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/21_fork_mojo.t ................... skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/30_bulk_add_action.t ............. skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/31_bulk_helpers.t ................ skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/32_bulk_flush.t .................. skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/33_bulk_errors.t ................. skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/34_bulk_cxn_errors.t ............. skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/40_scroll.t ...................... skipped: No \$ENV{CLIENT_VER} specified",
    "t/Client_6_0_Async/60_auth_aehttp.t ................. skipped: AEHTTP - No https server specified in ES_SSL",
    "t/Client_6_0_Async/61_auth_mojo.t ................... skipped: Mojo - No https server specified in ES_SSL",
    "",
    "Test Summary Report",
    "-------------------",
    "t/30_Logger_Async/40_trace_request.t              (Wstat: 1024 Tests: 11 Failed: 4)",
    "  Failed tests:  5, 7, 9, 11",
    "  Non-zero exit status: 4",
    "t/60_Cxn_Async/30_http.t                          (Wstat: 5376 Tests: 133 Failed: 21)",
    "  Failed tests:  1, 7, 13, 19, 25, 31, 37, 43, 49, 55, 61",
    "                67, 73, 79, 85, 91, 97, 103, 109, 115, 121",
    "  Non-zero exit status: 21",
    "Files=71, Tests=501, 23 wallclock secs ( 0.12 usr  0.22 sys + 16.69 cusr  5.73 csys = 22.77 CPU)",
    "Result: FAIL",
  ],
  via => "App::cpanminus::reporter 0.17 (1.7044)",
}
```

**Comment by ezimuel** [comment]
@jkeenan and @eserte I finally fixed this issue and also #183 with the PR #189. Please have a look, thanks!

**Comment by jkeenan** [comment]
> @jkeenan and @eserte I finally fixed this issue and also #183 with the PR #189. Please have a look, thanks!

For problems like this, I only test CPAN releases (not pull requests) because a tarball fetched from CPAN was the source of my report.  When do you anticipate doing a new CPAN release?

Thank you very much.
Jim Keenan

**Comment by ezimuel** [comment]
@jkeenan yes, I'll do a 6.81 release in the next days (this week). I want also to include this PR #190 in the release.

**Comment by eserte** [comment]
I tried `cd dist && dzil test` on the branch fix/182-183, and it passed.

**Comment by ezimuel** [comment]
Thanks @eserte!

**Comment by ezimuel** [comment]
Fixed with #189
